Sources/MapboxDirections/Intersection.swift

```diff
@@ -42,6 +46,10 @@ public struct Intersection: ForeignMemberContainer {
         self.restStop = restStop
         self.regionCode = regionCode
         self.outletMapboxStreetsRoadClass = outletMapboxStreetsRoadClass
+        self.railwayCrossing = railwayCrossing
+        self.trafficSignal = trafficSignal
+        self.stopSign = stopSign
+        self.yieldSign = yieldSign
```

Comment:
There’s a mix of American English and British English vocabulary here:

* American English: railroad crossing, yield sign
* British English: railway crossing, give way sign

We should be consistent in these names to reduce confusion. This package has been using American English spelling and vocabulary for the most part, except for the occasional oversight perhaps. But if “railroad crossing” is confusing or strange to an international audience, “level crossing” would be a decent compromise, as a more specific technical term.

Reply:
I didn't even think about it, just reflected original names from Directions. Thanks for clarification.